src/status_im2/contexts/activity_center/notification/admin/view.cljs

```diff
@@ -8,14 +8,31 @@
             [utils.i18n :as i18n]
             [utils.re-frame :as rf]))
 
+(defn swipeable
+  [{:keys [height active-swipeable notification]} child]
+  (case (:membership-status notification)
```

Comment:
why case? there will be more cases?

Reply:
Pure personal preference @flexsurfer. Without the `case` I'm forced to extract the value to a binding or repeat `(:membership-status notification)` twice.

```clojure
(defn swipeable
  [{:keys [height active-swipeable notification]} child]
  (let [membership-status (:membership-status notification)]
    (if (or (= membership-status constants/activity-center-membership-status-accepted)
            (= membership-status constants/activity-center-membership-status-declined))
      [common/swipeable
       {:left-button      common/left-swipe-button
        :left-on-press    common/left-swipe-on-press
        :right-button     common/right-swipe-button
        :right-on-press   common/right-swipe-on-press
        :active-swipeable active-swipeable
        :extra-fn         (fn [] {:height @height :notification notification})}
       child]
      child)))
```

Reply:
```clojure
(if (#{constants/activity-center-membership-status-accepted)
          constants/activity-center-membership-status-declined}
       membership-status ))
```
usually set is used for such cases

Reply:
Indeed, I actually forgot of this pattern. Thanks

Reply:
https://github.com/status-im/status-mobile/pull/15106/commits/4aae89a498a8dd2a74cd3eff895a16f401c94c03